HellBrick.NoCapture.Test/NoCaptureTest.cs

```diff
@@ -288,5 +288,195 @@ class C
 "
 			)
 			.ShouldHaveDiagnostic( "Invoke", "transforms", "this" );
+
```

Comment:
Let's get rid of an extra endline here =)

Reply:
Done =)